WordPress/Classes/Utility/FeatureFlag.swift

```diff
@@ -0,0 +1,74 @@
+/// FeatureFlag exposes a series of features to be conditionally enabled on
+/// different builds.
+@objc
+enum FeatureFlag: Int {
+    /// My Sites > Site > People
+    /// Development on hold while we focus on Me
+    case People
+    /// Me > My Profile
+    /// Needs better UI for loading/failed states
+    case MyProfile
+
+    /// Returns a boolean indicating if the feature is enabled
+    var enabled: Bool {
+        switch self {
+        case .People:
+            return build(.Debug)
+        case .MyProfile:
+            return build(.Debug, .Alpha, .Internal)
+        }
+    }
+}
+
+/// Objective-C bridge for FeatureFlag.
+///
+/// Since we can't expose properties on Swift enums we use a class instead
+class Feature: NSObject {
+    /// Returns a boolean indicating if the feature is enabled
+    static func enabled(feature: FeatureFlag) -> Bool {
+        return feature.enabled
+    }
+}
+
+/// Represents a build configuration.
+enum Build: Int {
+    /// Development build, usually what you get when you run from Xcode
+    case Debug
+    /// Daily buiilds released internally for Automattic employees
+    case Alpha
+    /// Beta released internally for Automattic employees
+    case Internal
+    /// Production build released in the app store
+    case AppStore
+
+    /// Returns the current build type
+    static var current: Build {
+        if let override = _overrideCurrent {
+            return override
+        }
+
+        #if DEBUG
```

Comment:
I think that bit wasn't clear. The build type is still defined by build parameters, but using this code does the check at runtime. For instance:

```swift
#if DEBUG
  doStuff() // this isn't included in the final binary
#endif
```

```swift
if build(.Debug) {
  doStuff() // this is included in the binary, but it's never reached
}
```